src/Drupal/Commands/core/MigrateRunnerCommands.php

```diff
@@ -459,6 +494,18 @@ protected function executeMigration(MigrationInterface $migration, string $migra
         $executedMigrations[] = $migrationId;
     }
 
+    /**
+     * Process options for rollback().
+     *
+     * @hook init migrate:rollback
+     * @option migrationList The array of migrations to process, used internally.
+     */
+    public function initRollback(InputInterface $input, AnnotationData $annotationData)
+    {
+        // Do the same as hook init migrate:import.
+        $this->initImport($input, $annotationData);
```

Comment:
This is ugly. I wonder if is not possible to have a single method, properly renamed, for both commands. I'm reading from https://github.com/consolidation/annotated-command#hooks

> An annotation (e.g. @foo) will attach the hook to any command that is annotated with the given label.

Could we explore this option?